src/iroc_bridge.cpp

```diff
@@ -571,39 +744,79 @@ void IROCBridge::parseGeneralRobotInfo(mrs_robot_diagnostics::GeneralRobotInfo::
 /* parseStateEstimationInfo() //{ */
 
 void IROCBridge::parseStateEstimationInfo(mrs_robot_diagnostics::StateEstimationInfo::ConstPtr state_estimation_info, const std::string& robot_name) {
-  const json json_msg = {
-      {"robot_name", robot_name},
-      {"estimation_frame", state_estimation_info->header.frame_id},
-      {"local_pose",
-       {{"x", state_estimation_info->local_pose.position.x},
-        {"y", state_estimation_info->local_pose.position.y},
-        {"z", state_estimation_info->local_pose.position.z},
-        {"heading", state_estimation_info->local_pose.heading}}},
-      {"global_pose",
-       {{"latitude", state_estimation_info->global_pose.position.x},
-        {"longitude", state_estimation_info->global_pose.position.y},
-        {"altitude", state_estimation_info->global_pose.position.z},
-        {"heading", state_estimation_info->global_pose.heading}}},
-      {"above_ground_level_height", state_estimation_info->above_ground_level_height},
-      {"velocity",
-       {{"linear",
-         {{"x", state_estimation_info->velocity.linear.x}, {"y", state_estimation_info->velocity.linear.y}, {"z", state_estimation_info->velocity.linear.z}}},
-        {"angular",
-         {{"x", state_estimation_info->velocity.angular.x},
-          {"y", state_estimation_info->velocity.angular.y},
-          {"z", state_estimation_info->velocity.angular.z}}}}},
-      {"acceleration",
-       {{"linear",
-         {{"x", state_estimation_info->acceleration.linear.x},
-          {"y", state_estimation_info->acceleration.linear.y},
-          {"z", state_estimation_info->acceleration.linear.z}}},
-        {"angular",
-         {{"x", state_estimation_info->acceleration.angular.x},
-          {"y", state_estimation_info->acceleration.angular.y},
-          {"z", state_estimation_info->acceleration.angular.z}}}}},
-      {"current_estimator", state_estimation_info->current_estimator},
-      {"running_estimators", state_estimation_info->running_estimators},
-      {"switchable_estimators", state_estimation_info->switchable_estimators}};
+
+  // Create the deeply nested JSON structure using Crow
+  crow::json::wvalue json_msg;
+
+  // Basic fields
+  json_msg["robot_name"] = robot_name;
+  json_msg["estimation_frame"] = state_estimation_info->header.frame_id;
+  json_msg["above_ground_level_height"] = state_estimation_info->above_ground_level_height;
+  json_msg["current_estimator"] = state_estimation_info->current_estimator;
+
+  // Local pose object
+  crow::json::wvalue local_pose;
+  local_pose["x"] = state_estimation_info->local_pose.position.x;
+  local_pose["y"] = state_estimation_info->local_pose.position.y;
+  local_pose["z"] = state_estimation_info->local_pose.position.z;
```

Comment:
It would be interesting to try this kind of initialization:
```c++
crow::json::wvalue data = {
  {"name", "John"},
  {"age", 30},
  {"address", {
    {"av", "Calle Falsa 123"},
    {"city", "Springfield"},
    {"country", "USA"}
  }},
  {"phones", crow::json::wvalue::list({
    {
      {"type", "home"},
      {"number", "123456789"}
    },
    {
      {"type", "personal"},
      {"number", "987654321"}
    }
  })}
}
```